guides/v2.3/graphql/reference/categories.md

```diff
@@ -34,13 +34,32 @@ Attribute | Data type | Description
 `path_in_store`| String | Category path in the store
 `url_key`| String | The url key assigned to the category
 `url_path`| String | The url path assigned to the category
-`is_active`| Boolean | Indicates whether the category is enabled
 `position`| Int | The position of the category relative to other categories at the same level in tree
 `level` | Int | Indicates the depth of the category within the tree
 `created_at`| String | Timestamp indicating when the category was created
 `updated_at`| String | Timestamp indicating when the category was updated
 `product_count`| Int | The number of products in the category
 `default_sort_by`| String | The attribute to use for sorting
+`products(<attributes>) | `CategoryProducts` | The list of products assigned to the category
+`children` | `CategoryTree` |
+
+The `products` attribute can contain the following attributes:
+
+Attribute | Data type | Description
+--- | ---
+`pageSize` | Int | Specifies the maximum number of results to return at once. This attribute is optional. The default value is 20.
+`currentPage` | Int |  Specifies which page of results to return. The default value is 1.
```

Comment:
Is `currentPage` also optional? I just ask because it also has a default value.

Reply:
Technically, it's not required, but it's hard to imagine anyone not using it to determine where they are in the list of returned results